Makefile

```diff
@@ -209,11 +212,14 @@ deploy-service-requirements: deploy-namespace deploy-inventory-service-file
 deploy-resources: generate-manifests
 	python3 ./tools/deploy_crd.py $(ENABLE_KUBE_API_CMD) --apply-manifest $(APPLY_MANIFEST) --profile "$(PROFILE)" --target "$(TARGET)"
```

Comment:
I don't see anyone calls `deploy-resources` except your deletions. I'm not familiar with its goal... Could it be removed?